src/main/asciidoc/index.adoc

```diff
@@ -469,4 +469,16 @@ stream of {@link org.apache.kafka.clients.producer.ProducerRecord}.
 The API exposed by these interfaces is mostly the same than the polyglot version.
 endif::[]
 
+== Automatic trace propagation
+
+When Vert.x is configured with Tracing enabled (see {@link io.vertx.core.VertxOptions#setTracingOptions}),
```

Comment:
I think we need to tell give here the list of tags that are propagated even though we reference the open tracing spec

Reply:
just did it